src/assets/i18n/es.json

```diff
@@ -0,0 +1,245 @@
+      "PRICE": "Precio",
+      "VOLUME": "Volumen"
+    },
+    "NAME": "Nombre",
+    "NETWORKS_PAGE": {
+      "CHANGE_PEER": "Cambiar de Peer",
+      "CHOOSE_ONE": "Elija uno de los tipos de red",
+      "DELAY": "Retraso",
+      "HEIGHT": "Altura",
+      "LOOKING_GOOD_PEER": "Buscando un buen peer",
+      "PEER_SUCCESSFULLY_CHANGED": "Peer cambiado con éxito",
+      "NETWORK_STATUS": "Estado de red",
+      "NETWORK_TYPE": "Tipo de red",
+      "INTERNET_DESCONNECTED": "No se pudo conectar a Internet",
+      "PEER": "Peer"
+    },
+    "NEXT": "Próximo",
+    "NO": "Não",
+    "PASSPHRASE_TEST": {
+      "INFO": "Por favor, introduzca las palabras de su frase secreta para validar la creación de la cartera.",
+      "WRONG_WORD": "La palabra introducida es incorrecta"
+    },
+    "PIN_CODE": {
+      "CONFIRM": "Confirme su PIN",
+      "CREATE": "Crear su PIN",
+      "DEFAULT_MESSAGE": "Introduzca su PIN",
+      "WRONG_PIN_MANY_TIMES": "¡Usted ha informado de un PIN incorrecto varias veces!",
+      "WAIT_TO_TRY": "Por favor espere {{ time }} y vuelva a intentarlo.",
+      "SIGN_IN_ERROR": "Ha habido un problema al iniciar sesión con su PIN",
+      "PIN_CREATED_ERROR_TEXT": "Ha habido un problema al crear su PIN",
+      "PIN_CREATED_TEXT": "Su PIN se creó con éxito",
+      "PIN_UPDATED_ERROR_TEXT": "Ha habido un problema al intentar actualizar su PIN",
+      "PIN_UPDATED_TEXT": "Su PIN se ha actualizado con éxito",
+      "TYPE_PIN_ENCRYPT_PASSPHRASE": "Introduzca su PIN para encriptar la frase secreta",
+      "TYPE_PIN_SIGN_TRANSACTION": "Introduzca su PIN para firmar la transacción",
+      "WRONG": "Por favor, introduzca el PIN correcto"
+    },
+    "PROFILES_PAGE": {
+      "ADD_PROFILE_ERROR": "No se pudo agregar el perfil",
+      "ADD_PROFILE_TEXT": "Haga clic en el botón de abajo para crear un perfil",
+      "CREATE_PROFILE": "Crear perfil",
+      "DELETE_NOT_EMPTY": "No puede borrar un perfil que tiene 1 o más carteras",
+      "GET_STARTED": "Comienzo",
+      "SIGN_IN": "Registro"
+    },
+    "QR_CODE": {
+      "INVALID_QR_ERROR": "Ha habido un problema al intentar escanear el Código QR",
+      "INVALID_QRCODE": "Este código QR no tiene una dirección válida o passphrase",
+      "PERMISSION_DENIED": "Permiso para la cámara QR negada",
+      "PERMISSION_PERMANENTLY_DENIED": "Permiso para la cámara QR permanentemente negada. Vaya a la configuración de su dispositivo para cambiar.",
+      "PROBLEM_TEXT": "Ha habido un problema al escanear el Código QR"
+    },
+    "RENAME": "Rebautizar",
+    "SAVE": "Guardar",
+    "SEARCH": "Buscar",
+    "SETTINGS_PAGE": {
+      "ABOUT": "En",
+      "APPEARANCE": "Apariencia",
+      "APPLICATION": "Solicitud",
+      "CHANGE_PIN": "Cambiar PIN",
+      "CLEAR_DATA": "Borrar datos",
+      "CLEAR_DATA_TEXT": "Todos los datos, incluidos los perfiles creados, las carteras y los contactos se eliminan de la aplicación y se restablecerá a la predeterminada.",
```

Comment:
" ... se eliminarán ... reestablecerán los predeterminados"

Reply:
Not apply.

line not informed.

content out of context.

Reply:
review use contect:

https://github.com/ArkEcosystem/desktop-wallet/blob/master/po/es-419.po